src/test/java/teammates/test/cases/BaseTestCaseWithBackDoorApiAccess.java

```diff
@@ -10,26 +10,55 @@
 import teammates.common.datatransfer.attributes.FeedbackSessionAttributes;
 import teammates.common.datatransfer.attributes.InstructorAttributes;
 import teammates.common.datatransfer.attributes.StudentAttributes;
+import teammates.common.util.Const;
 import teammates.test.driver.BackDoor;
+import teammates.test.driver.retry.RetryManager;
+import teammates.test.driver.retry.RetryableTaskReturns;
 
 /**
  * Base class for all test cases which are allowed to access the Datastore via {@link BackDoor}.
  */
 public abstract class BaseTestCaseWithBackDoorApiAccess extends BaseTestCaseWithDatastoreAccess {
 
+    protected AccountAttributes getAccount(String googleId) {
```

Comment:
I guess I'll leave those methods there first, then refactor in another issue and PR?